pyproject.toml

```diff
@@ -0,0 +1,126 @@
+[project]
+name = "pygeoapi"
+version = "0.14.dev0"
+description = "pygeoapi provides an API to geospatial data"
+keywords = ["geospatial data api"]
+readme = "README.md"
+authors = [
+    {name = "Tom Kralidis", email = "tomkralidis@gmail.com"},
+]
+maintainers = [
+    {name = "Tom Kralidis", email = "tomkralidis@gmail.com"},
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Topic :: Scientific/Engineering :: GIS",
+]
+dependencies = [
```

Comment:
Many dependencies are in twice, e.g. `pydantic`, some with and without pinning like `jinja2` and `jinja2==3.0.3`. A copy/paste issue or am I missing something?

Reply:
I think I converted the project with pdm and then imported all the requirements files again, but the basic one were there already.

Reply:
But still I don't understand why many packages are in `dependencies[]`  twice even pinned and unpinned (viz `jinja2`. The order is also not significant, usually alphabetic, ignoring upper/lowercase. I mean like Babel, pydantic etc. we need these only once, or is the order somehow here significant?